packages/atlas-service/src/main.ts

```diff
@@ -46,8 +47,10 @@ const redirectRequestHandler = oidcServerRequestHandler.bind(null, {
 /**
  * https://www.mongodb.com/docs/atlas/api/atlas-admin-api-ref/#errors
  */
-function isServerError(err: any): err is { errorCode: string; detail: string } {
-  return Boolean(err.errorCode && err.detail);
+function isServerError(
+  err: any
+): err is { error: number; errorCode: string; detail: string } {
+  return Boolean(err.error && err.errorCode && err.detail);
```

Comment:
Pretty sure this line will break the tests because we are expecting a different error schema from the backend now

Reply:
Yeah, did not run tests locally let CI do it. Fixed the test. Hopefully, now it should be green.